test/run_e2e_test.sh

```diff
@@ -51,14 +51,15 @@ setup(){
   temp_path=$(mktemp -d)
   mkdir "$temp_path/sysroot"
   mkdir -p "$temp_path/sysroot/ros2_ws/src/"
-  wget https://raw.githubusercontent.com/ros2/ros2/dashing/ros2.repos --directory-prefix="$temp_path/sysroot"
-  vcs import "$temp_path/sysroot/ros2_ws/src" < "$temp_path/sysroot/ros2.repos"
-  mkdir -p "$temp_path/sysroot/qemu-user-static"
-  cp /usr/bin/qemu-* "$temp_path/sysroot/qemu-user-static"
   # Get full directory name of the script no matter where it is being called from
   local dir
   dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-  cp "$dir/Dockerfile_workspace" "$temp_path/sysroot/"
+  # Copy custom ROS2 Dashing deps
+  cp "$dir/ros2_test.repos" "$temp_path/sysroot"
+  vcs import "$temp_path/sysroot/ros2_ws/src" < "$temp_path/sysroot/ros2_test.repos"
+  # Copy QEMU binaries
+  mkdir -p "$temp_path/sysroot/qemu-user-static"
+  cp /usr/bin/qemu-* "$temp_path/sysroot/qemu-user-static"
```

Comment:
quote? does this still pass shellcheck?

Reply:
Yes, if you add quotes the wildcard doesn't copy the binaries.
This was addressed in a previous comment.